Core/Utilities/test/Test_ProcessPool.py

```diff
@@ -24,18 +24,30 @@
 
 ## imports 
 import os
+import sys
 import unittest
 import random
 import time
 import threading
 
+import pytest
+
 ## from DIRAC
 # from DIRAC.Core.Base import Script
 # Script.parseCommandLine()
 from DIRAC import gLogger
 ## SUT
 from DIRAC.Core.Utilities.ProcessPool import ProcessPool
 
+
+@pytest.fixture(autouse=True)
+def capture_wrap():
+  """ Avoid https://github.com/pytest-dev/pytest/issues/5502 """
+  sys.stderr.close = lambda *args: None
+  sys.stdout.close = lambda *args: None
```

Comment:
ouch :D 
Though I learned something about `autouse` :-)

Reply:
It's actually still a mess as DIRAC's logging prevents the stderr messages from being captured by each test. What actually happens is `pytest` attributes all of the output to the first test that is executed. (That was a fun 20 minutes to make sense of...)

Reply:
The more I think of it, the more I feel like scrapping gLogger altogether. Although I suspect base logging would do the same no ?

Reply:
I think the base logging would do it slightly differently as I've used it in other places and not had this issue (but I didn't look into it properly).